docs/running.md

```diff
@@ -0,0 +1,209 @@
+---
+layout: doc
+---
+
+# Running ImapTest
+
+## Parameters (Primary)
+
+### Parameter: `host`
+
+* Default: `127.0.0.1`
+
+Host name/IP address where to connect.
+
+### Parameter: `port`
+
+* Default: `143`
+
+Port to connect to.
+
+### Parameter: `user`
+
+* Default: `$USER`
+
+Username template.
+
+You can use multiple random users and domains by giving `%d` in the template. So for example `user%d` returns `user1..user100` and `user%d@domain%d.org` returns `user1..100@domain1..100.org`.
+
+The upper limit for users can be set with the [`users`](#parameter-users) parameter, and the upper limit for domains can be set with the [`domains`](#parameter-domains) parameter.
+
+### Parameter: `userfile`
+
+* Default: \<none\>
+
+Read usernames from given file, one user per line. It's also possible to give passwords for users in `username:password` format.
+
+### Parameter: `pass`
+
+* Default: \<none\>
+
+Password to use for all users.
+
+::: warning
+There's currently no way to use different passwords for different users.
```

Comment:
it is with userfile

Reply:
Clarified that this comment applies only to command-line password option, and point at userfile option